docs/DeveloperGuide.md

```diff
@@ -605,32 +648,104 @@ Similar to [<ins>UC07](#use-case-uc07-add-phone-number-to-contact) except duplic
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by wedding.
+### **Use case: UC16 — Search for Contacts by Wedding**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by wedding.
 
-### **Use case: UC13 — Search for Contacts by Phone Number**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by phone number.
+### **Use case: UC17 — Search for Contacts by Phone Number**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by phone number.
 
-### **Use case: UC14 — Search for Contacts by Address**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by address.
+### **Use case: UC18 — Search for Contacts by Address**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by address.
 
-### **Use case: UC15 — Search for Contacts by Email Address**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by email address.
+### **Use case: UC19 — Search for Contacts by Email Address**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by email address.
 
 ---
 
-### **Use case: UC16 — Search for Wedding by Wedding Name**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for wedding by wedding name.
+### **Use case: UC20 — Search for Wedding by Wedding Name**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for wedding by wedding name.
 
-### **Use case: UC17 — Search for Task by Task Name**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for task by task name.
+### **Use case: UC21 — Search for Task by Task Name**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for task by task name.
 
 ---
 
-### **Use case: UC18 — Delete Contact**
+### **Use case: UC22 — Assign a Contact to one or more Weddings**
+
+**MSS**
+
+1. User <ins>lists all weddings (UC02)</ins>.
+2. User requests to assign a contact to a wedding.
+3. System displays the contact being associated with the specified wedding, in the list of contacts.
+
+**Extensions**
+
+* 1a. The contact list is empty.
+  Use case ends.
+
+* 1b. The wedding list is empty.
+  Use case ends.
+
+* 2a. A specified wedding does not exist in the System.
+    * 2a1. The system displays an error message.
+
+      Use case resumes at step 1.
+
+### **Use case: UC23 — Unassign a Contact from one or more Weddings**
+Similar to [<ins>UC22](#use-case-uc22-assign-a-contact-to-one-or-more-weddings) except un-assigning a contact from Weddings. Additionally, for **Extension 2a.** it checks to ensure the specified wedding is currently associated with the contact, instead of simply existing in the system.
```

Comment:
I like the inclusion of the extra check - am very unsure how to format this, would plain text be okay? Or should it be more of a 2b. ?

Reply:
im also not too sure because it just checks if the specified wedding is currently associated with the contact then the 2a scenario to check if it exists might not happen since it kind of only depends if the wedding is associated with the contact. But for now, i will add it as a addition extension 2b. and we can decide after how it should be